.circleci/config.yml

```diff
@@ -277,6 +277,7 @@ jobs:
           channel: "C7H40L71D" # dsde-qa-notify
   integration-tests-pr-staging:
     executor: puppeteer
+    resource_class: medium+
```

Comment:
Had to go to a larger resource class because the `npx serve build` step kept failing with 'Received "killed" signal'.
https://app.circleci.com/pipelines/github/DataBiosphere/terra-ui?branch=node-20